Assets/Scripts/InventoryScripts/JournalInventoryScript.cs

```diff
@@ -1,147 +1,28 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using Dialogue_scripts;
 using UnityEngine;
 using UnityEngine.UI;
 
-public class JournalInventoryScript : MonoBehaviour {
-
-    public Button[] journalItemPlace;                //creates an array of Buttons
-    public Button[] pageItemPlace;
-    public GameObject[] GoodPages;
-    public GameObject[] EvilPages;
-    public GameObject[] DeceitfulPages;
-    public GameObject[] UnfortunatePages;
-    public ManagerScript manager;                    //reference to ManagerScript
+public class JournalInventoryScript : MonoBehaviour
+{
+    [SerializeField] private Button[] pageItemPlace;
+    [SerializeField] public GameObject[] GoodPages;
+    [SerializeField] public GameObject[] EvilPages;
+    [SerializeField] public GameObject[] DeceitfulPages;
+    [SerializeField] public GameObject[] UnfortunatePages;
     [SerializeField] private DeathMovement _thePlayer;
     [SerializeField] private Test _testPlayer;
-
-
-
-    // Use this for initialization
-    void Start() {
-
-    }
-
-    // Update is called once per frame
-    void Update() {
-
-    }
+    [SerializeField] public GameObject[] _books;
```

Comment:
public serialized field with private field naming conventions. Please pick one or the other and adjust.